platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/pipeline/VirtualPipeline.java

```diff
@@ -256,39 +256,55 @@ Duration calculateFlushBackpressurePause() {
         final Duration sleepTime = CompareTo.min(computedSleepTime, maxSleepTime);
         final int sleepTimeMillis = (int) sleepTime.toMillis();
         statistics.recordFlushBackpressureMs(sleepTimeMillis);
```

Comment:
Should statistics be updated every time we calculate a new value or only when we apply it (similar to `applyFamilySizeBackpressure()`)?
The two backpressure methods should be consistent in that regard.

Reply:
Good catch. I didn't realize the metrics were reported in the "calculate" method, not when the backressure is actually applied. With my current changes it was fixed for family size backpressure, but not for flush backpressure. I'll change it accordingly